lnrpc/rpc.proto

```diff
@@ -503,24 +503,29 @@ service Lightning {
     }
 
     /** lncli: `sendpayment`
-    SendPayment dispatches a bi-directional streaming RPC for sending payments
-    through the Lightning Network. A single RPC invocation creates a persistent
-    bi-directional stream allowing clients to rapidly send payments through the
-    Lightning Network with a single persistent connection.
-    */
-    rpc SendPayment (stream SendRequest) returns (stream SendResponse);
+    Deprecated, use routerrpc.SendPayment. SendPayment dispatches a
+    bi-directional streaming RPC for sending payments through the Lightning
+    Network. A single RPC invocation creates a persistent bi-directional
+    stream allowing clients to rapidly send payments through the Lightning
+    Network with a single persistent connection.
+    */
+    rpc SendPayment (stream SendRequest) returns (stream SendResponse) {
+        option deprecated = true;
+    }
 
     /**
-    SendPaymentSync is the synchronous non-streaming version of SendPayment.
-    This RPC is intended to be consumed by clients of the REST proxy.
-    Additionally, this RPC expects the destination's public key and the payment
-    hash (if any) to be encoded as hex strings.
+    Deprecated, use routerrpc.SendPayment. SendPaymentSync is the synchronous
+    non-streaming version of SendPayment. This RPC is intended to be consumed
+    by clients of the REST proxy. Additionally, this RPC expects the
+    destination's public key and the payment hash (if any) to be encoded as
+    hex strings.
     */
     rpc SendPaymentSync (SendRequest) returns (SendResponse) {
         option (google.api.http) = {
             post: "/v1/channels/transactions"
             body: "*"
         };
+        option deprecated = true;
```

Comment:
I think so. We only offer one way of sending a payment.

Reply:
The prior async version (async as being able to stream multiple payments with one call), was a very premature optimization imo. I haven't even been able to reproduce performance benefits.

Reply:
We could add a convenience rpc on `routerrpc` that blocks until there is a final outcome.

Reply:
Offline discussion: convenience method isn't enough, because also rest wrappers are needed on `routerrpc`. For now decided to only deprecated the async send payment call in this PR.